lisa/sut_orchestrator/libvirt/platform.py

```diff
@@ -95,6 +93,7 @@ def __init__(self, runbook: schema.Platform) -> None:
         self.platform_runbook: BaseLibvirtPlatformSchema
         self.host_node: Node
         self.vm_disks_dir: str
+        self.vm_disk_img: str
```

Comment:
Let's use `Path` objects?

Reply:
@anirudhrb I think its fine to have it as str. We dont do much Path operations here. The only usage of `vm_disk_img` is assigning appropriate value to `os_disk_base_file_path` which is str and is used as str in multiple places.